firmware/src/iot_config/IoT_Sensor_Node_config.h

```diff
@@ -0,0 +1,20 @@
+#ifndef IOT_SENSOR_NODE_CONFIG_H
+#define IOT_SENSOR_NODE_CONFIG_H
+
+#define CFG_DEFAULT_TELEMETRY_INTERVAL 10

```

Comment:
is this `_MS` or `_SEC`?
```suggestion
#define CFG_DEFAULT_TELEMETRY_INTERVAL 10

```

Reply:
Added _SEC